src/store/metadata/stub/content/datasets.json

```diff
@@ -234,6 +234,58 @@
         }
       ]
     },
+    {
+      "@id": "https://staging.idpd.uk/datasets/additions-to-the-housing-stock",
+      "@type": "dcat:DatasetSeries",
+      "identifier": "additions-to-the-housing-stock",
+      "title": "Net additions to the housing stock",
+      "summary": "Net additions to the dwelling stock (including new builds, conversions, changes of use, other gains/losses and offset by demolitions) per 1,000 stock, England, financial year 2022 - 2023.",
+      "description": "This dataset shows the net additions to the dwelling stock (including new builds, conversions, changes of use, other gains/losses and offset by demolitions) per 1,000 stock, in England, for financial year 2022 - 2023.\n\n### Coverage \n\nEngland\n\n#### Geography definition\n\nLocal Authority Districts and Unitary Authorities,\nCounties,\nMetropolitan Counties,\nNation (England) \n\n### Source\n\n - England: [Department for Levelling Up, Housing and Communities",
+      "issued": "2023-05-05T08:59:11Z",
+      "modified": "2023-12-06T15:10:00Z",
+      "next_release": "2024-04-19T09:30:00Z",
+      "publisher": "https://staging.idpd.uk/publishers/department-for-levelling-up-housing-and-communities",
```

Comment:
Please change to "https://staging.idpd.uk/publishers/office-for-national-statistics"